airflow/providers/google/cloud/example_dags/example_life_sciences.py

```diff
@@ -0,0 +1,106 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""
+Example Airflow DAG that displays interactions with Google Cloud Life Sciences.
+
+This DAG relies on the following OS environment variables:
+
+* GCP_PROJECT_ID - Google Cloud Project to use for the Cloud Function.
+* GCP_GCS_BUCKET - Google Cloud Storage Bucket to use
+* GCP_LOCATION - The Location of the Google Cloud Project
+"""
+import os
+
+from airflow import models
+from airflow.providers.google.cloud.operators.life_sciences import LifeSciencesRunPipelineOperator
+from airflow.utils import dates
+
+PROJECT_ID = os.environ.get("GCP_PROJECT_ID", "example-project-id")
+BUCKET = os.environ.get("GCP_GCS_BUCKET", "example-bucket")
+LOCATION = os.environ.get("GCP_LOCATION", 'example-location')
```

Comment:
```suggestion
LOCATION = os.environ.get("GCP_LIFE_SCIENCE_LOCATION", 'europe-west4')
```
Can you use unique name for variable? This is useful because we have many system tests. The default value that may work is also useful because it allows you to run tests with fewer environment variables.

Reply:
Ok. Thanks